notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<p>I would add a markdown note about training time here. With a T4 it took about 7 minutes to train 10 epochs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Makes sense. Added that line! Thanks!</p>